src/Toolkit/V.php

```diff
@@ -239,13 +239,45 @@ public static function __callStatic(string $method, array $arguments): bool
     },
 
     /**
-     * Checks for a valid date
-     */
-    'date' => function ($value): bool {
-        $date = date_parse($value);
-        return $date !== false &&
-                $date['error_count'] === 0 &&
-                $date['warning_count'] === 0;
+     * Checks for a valid date and is able to compare
+     * dates against each other.
+     *
+     * Pass only the first argument to check for a valid date.
+     * Pass an operator as second argument and another date as
+     * third argument to compare them.
+     */
+    'date' => function ($value, string $operator = null, $test = null): bool {
+        $args = func_get_args();
+
+        // simple date validation
+        if (count($args) === 1) {
+            $date = date_parse($value);
+            return $date !== false &&
+                    $date['error_count'] === 0 &&
+                    $date['warning_count'] === 0;
+        }
+
+        $value = strtotime($value);
+        $test  = strtotime($test);
+
+        if (is_int($value) !== true || is_int($test) !== true) {
+            return false;
+        }
+
+        switch ($operator) {
+            case '!=':
+                return $value != $test;
```

Comment:
```suggestion
                return $value !== $test;
```

The same applies to `$value == $test`, which would need to be `$value === $test`.